test.md

```diff
@@ -0,0 +1 @@
+"This file should be ignored by git"
```

Comment:
- [ ] You did a great job following the requirements! 💯  However, please make sure that this file is ignored by git. Consider fixing this one! 🤓 🛠️